content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -42,149 +40,148 @@ To configure AD as the IdP, you must create and modify a Relying Party Trust. Af
+   d. Enter the name for the Relying Party Trust in the **Display Name** field. For this example, we use *QE SAML*. QE is the Windows administrator name, and SAML is the service name.
       
       ![WizardTypeDisplayNameCropped](/images/SAML/WizardTypeDisplayNameCropped.png "Wizard Type Display Name Cropped")
    
-   e. Click **Next**. The Configure Certificate window displays.
+   e. Click **Next** to move on to the **Configure Certificate** window.
 
-3. Modify the TrueCommand Certificate (tc.cer).
+3. Modify the TrueCommand Certificate (<file>tc.cer</file>).
 
-   a. Open PowerShell and type `Invoke -webRequest -uri http://IP:PORT/saml/metadata -outfile tc.cer` where IP:PORT is your TrueCommand system’s IP address/port number.
+   a. Open PowerShell and type `Invoke -webRequest -uri http://IP:PORT/saml/metadata -outfile tc.cer` where *IP:PORT* is your TrueCommand system IP address/port number.
 
       ![OpenPowershellTypeInvokeCommand](/images/SAML/OpenPowershellTypeInvokeCommand.png "Open Powershell Type Invoke Command")
 
    b. Edit the certificate as follows:
 
-      1. Open a file Explorer window and locate the tc.cer file in c:/local data/user/QE.
+      i. Open a **File Explorer** window and locate the <file>tc.cer</file> file in **C:/local data/user/QE**.
     
          ![LocateTcCertificateWithFileExplorer](/images/SAML/LocateTcCertificateWithFileExplorer.png "Locate TC Certificate With File Explorer")
         
-      2. Select the tc.cer file, right click and select Open with Notepad.
-      3. Delete everything before the <x509Certificate> tag, and everything after the </x509Certificate> tag.
+      ii. Select the <file>tc.cer</file> file, right-click, then select **Open with Notepad**.
+      iii. Delete everything before the `<x509Certificate>` tag, and everything after the `</x509Certificate>` tag.
 
          ![OpenTcCertificate](/images/SAML/OpenTcCertificate.png "OpenTcCertificate")
         
-      4. Type the following string exactly as `-----BEGIN CERTIFICATE-----` before the certificate with five dashes before and after..
-      5. Type the following string exactly as `-----END CERTIFICATE-----` after the certificate with five dashes before and after.
+      iv. Type the following string exactly as `-----BEGIN CERTIFICATE-----` before the certificate with five dashes before and after.
+      v. Type the following string exactly as `-----END CERTIFICATE-----` after the certificate with five dashes before and after.
       
          ![EditedTcCertificate](/images/SAML/EditedTcCertificate.png "Edited Tc Certificate")
       
-      6. Select Save (or Ctrl-S) and then close Notepad.
+      vi. Click **Save** (or <kbd>Ctrl-S</kbd>) and then close **Notepad**.
 
    c. Close Powershell.
 
    d. Select Browse on the Configure Certificate window. The Encryption Certificate window opens. 
```

Comment:
Bold is missing:

   d. Select **Browse** on the **Configure Certificate** window. The **Encryption Certificate** window opens.

Reply:
Fixed